man/docker-inspect.1.md

```diff
@@ -24,8 +24,85 @@ each result.
 **-f**, **--format**=""
     Format the output using the given go template.
 
+**--type**=*container*|*image*
+    Return JSON for specified type, permissible values are "image" or "container"
+
 # EXAMPLES
 
+## Getting information on an image where image name conflict with the container name.
+## e,g both image and container are named rhel7.
+
+   $ docker inspect --type=image rhel7
+   [
```

Comment:
I wonder if we need a full output example in this case to demonstrate this flag, because a full example of the output is already included below and is not the important bit here. I would be okay with "cutting" the output, e.g.

```
[
{
  "Id": "fe01a428b9d9de35d29531e9994157978e8c48fa693e1bf1d221dffbbb67b170",
  "Parent": "10acc31def5d6f249b548e01e8ffbaccfd61af0240c17315a7ad393d022c5ca2"
  ....
```

It also looks like the indentation is not 4 spaces, causing it not to be rendered as a code-block (see https://github.com/shishir-a412ed/docker/blob/docker_inspect_issue/man/docker-inspect.1.md)